src/main/java/org/balanpy/ConfiguracionMascota.java

```diff
@@ -0,0 +1,92 @@
+package org.balanpy;
+
+import java.net.URL;
+import java.time.LocalDate;
+import java.util.ResourceBundle;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+
+public class ConfiguracionMascota implements Initializable{
+	
+	@FXML
+	TextField nombre_mascota;
```

Comment:
Como mencioná a algún otro compañero, en Java utilizamos camelCase para miembros de una clase. Es una cuestión menor de consistencia. También habrá que cambiarlos en la pantalla.
```suggestion
	TextField nombreMascota;
```